docs/api.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: API
+---
+
+<!-- Warning: AUTO-GENERATED file, do not edit. Add code comments to your Ruby instead <3 -->
+
+# API
```

Comment:
I like this, but we do miss out on a lot of functionality that the YARD generated documentation pages provide like viewing the source inline, links to related code (like exceptions), search, etc.

What if we added a top-level documentation link to https://rubydoc.info/gems/view_component?

Reply:
@BlakeWilliams I hear you. It'd be easy to add inline source, and we already have search built into the docs site.

More generally, I see some downsides to using RubyDoc:

- Users would have to navigate between two sites.
- Anecdotally, RubyDoc is very slow to load for me.
- We'd be giving up control over how our API docs are organized. For example, the Collections feature exists in a module, but it's included automatically. Should the consumer of the docs have to navigate this complexity, knowing to look in `ViewComponent::Collection`? I personally would rather have a single page for all of the public methods that are available by default, regardless of where they come from.